src/Uno.UI.Tests/App/Views/BooleanToBrushConverter.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Media;
+
+namespace Uno.UI.Tests.App.Views
+{
+	public class BooleanToBrushConverter : IValueConverter
+	{
+		public Brush TrueValue { get; set; }
+		public Brush FalseValue { get; set; }
+
+		public object Convert(object value, Type targetType, object parameter, string language)
+		{
+			if (value is bool b)
+			{
+				return b ? TrueValue
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this condition so that it does not always evaluate to 'true'; some subsequent code is never executed.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305744)